### DISCUSSION TITLE
EP is crashing on today's latest Windows 11 updates.

### DISCUSSION THREAD

**Discussion by RobertLaffan** [valinet/ExplorerPatcher#1702] [General]
I will try to get version numbers, etc. and add them here. This is just a heads-up.

**Comment by giannikis** [comment]
Happens to me since yesterday.
I tried resetting all EP settings to default without luck.

Running EP 22621.1992.56.1
Windows 11 Pro 22621.2070

**Reply by aldeniangman** [reply]
i agree on 25931 i can't still change the taskbar position or have small icons although it says.

**Comment by pyrates999** [comment]
That is a preview update: https://support.microsoft.com/en-us/topic/july-26-2023-kb5028254-os-build-22621-2070-preview-66161ea6-c106-43e9-bef5-8410ccbd34e8

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

**Comment by giannikis** [comment]
Thanks @pyrates999 
It's weird. I did not manually run windows update. Did microsoft start forcing preview updates to end-users?

**Comment by pyrates999** [comment]
Microsoft currently does not automatically install preview updates.  They show a prompt asking you to download and install it.  Preview updates also come out during the second half of a month.  Don't install them if you want to use explorer patcher.

They're also called preview updates for a reason in that they're not fully tested.

**Comment by tpcr** [comment]
Uninstalling this preview is not in my uninstall list. I will have to uninstall EP until an update is available

**Comment by vippium** [comment]
To fix crashing on preview updates, simply uninstall EP completely and delete these reg. keys :
`Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher`
`Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher`

After deleting these, restart ur device. Then, install EP (with latest updates) and wait till the symbols to be downloaded msg appear. After that, simply restart ur device again.
**For win11 flyout users, action center will be overlapped over Win10 taskbar.**

**Reply by tpcr** [reply]
Those steps seem to work for me. I started the EP console and it went through updates a few times then finished. After that EP is working again.
Thanks!!

**Reply by PMcKenzie2004** [reply]
Thanks, this worked for me.

**Reply by BenjaminWKI** [reply]
Thank you. I can confirm this fix has my EP back to normal without needing me to uninstall any of my Windows Updates.

**Reply by YDITC** [reply]
Didn't work for me. I uninstalled EP, remved both registry folders, restarted, installed the latest version, waited for the symbols update, restarted again, and still, right-clicking the win11 taskbar just restarts the explorer 🤷🏻‍♂️
WinVer 22621.2070

**Reply by tpcr** [reply]
I have performed these instructions on 6 machines without a problem and working fine. You must have some other issue. Try another profile.

**Reply by ghost** [reply]
I have this same issue and the fix didn't work for me. However, I noticed that this issue does not happen when the Windows 10 style taskbar is selected it's only with the Windows 11 style taskbar. Just thought I'd mention it for what it's worth.

**Reply by IMMX5** [reply]
This worked for me  - THANX!!!

These were the Win11 updates installed today
2023-08 Cumulative Update for Windows 11 Version 22H2 for x64-based Systems (KB5029263)
and
2023-08 Cumulative Update for .NET Framework 3.5 and 4.8.1 for Windows 11, version 22H2 for x64 (KB5028948)

A shame I didn't see this fix 2 weeks ago - would have saved some anguish on my part!!

**Reply by dthoeni** [reply]
Great solution. Works fine!
Thanks for sharing.

**Reply by Leokuma** [reply]
Worked for me, but after reinstalling EP I also had to change some (any) settings and click on Restart File Explorer. I also switched my theme from dark to light a couple of times and eventually it stabilized apparently.

**Reply by pronode** [reply]
Worked for me!

**Reply by spiralofhope** [reply]
My solution didn't need a second reboot:

1. Uninstall ExplorerPatcher
2. Launch `regedit.exe`
3. Search for `Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher`
4. Delete that folder and its sub-folders.
5. Reboot
6. Reinstall ExplorerPatcher. It will take some time, but it will work with all your earlier settings.

**Reply by pyrates999** [reply]
I include a reboot due to finding that sometimes after the symbols have been downloaded, and you restarted file explorer, some applications were crashing when I tried to run them.  Only a reboot fixed that I found.

**Comment by pyrates999** [comment]
In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

**Comment by BinToss** [comment]
If the taskbar only crashes when you right-click it, then you might not need to uninstall EP.
In ExplorerPatcher Properties, go to Advanced and tick "Do not change the taskbar context menu (...)".

I'm running Win11 22H2 (22621.2070) and EP v22621.1992.56.1

**Reply by tpcr** [reply]
If explorer crashes when you right click it, how are you going to get to the EP properties? Is there another way? Maybe a reg patch.

**Reply by BinToss** [reply]
> If explorer crashes when you right click it, how are you going to get to the EP properties? Is there another way? Maybe a reg patch.

Search for "ExplorerPatcher" or "Properties" in the Start/Search menu.

**Comment by pyrates999** [comment]
I would just uninstall the preview update and don't install preview updates in the future.

**Reply by BinToss** [reply]
~~Cumulative Update previews typically do not have the option of being uninstalled or rolled back.~~

EDIT: Nevermind. It's hidden away in a different page and it will have a different name in that page.
Instead of `2023-07 Cumulative Update Preview for Windows 11 Version 22H2 for x64-based Systems (KB5028254)`, look for `Update for Microsoft Windows (KB5028254)`.

![image](https://github.com/valinet/ExplorerPatcher/assets/7243190/b9586708-ce55-46f3-b41d-5ed7837d9bf4)

**Reply by pyrates999** [reply]
Sure they do.  https://answers.microsoft.com/en-us/windows/forum/all/can-you-uninstall-a-cumulative-update-preview/8e5294b6-a398-4296-a395-e42e7ef9e011

**Reply by tpcr** [reply]
Not so. The list of updates are there but there is no option to uninstall them.
![updates1](https://github.com/valinet/ExplorerPatcher/assets/22845650/f828c270-ef0c-45de-9979-dba4dbb11bc7)

**Reply by BinToss** [reply]
There's no Uninstall option when you right-click the update, either.

**Reply by tallman69** [reply]
I have the latest version of Windows 11.  If I scroll down to the bottom of the Update history page under `Related settings` there is a setting titled `Uninstall updates`.  When I click on that it will display which updates are available to uninstall.

**Reply by BinToss** [reply]
Ohhhh! The problematic Cumulative Update Preview has a *different name* in the `Uninstall updates` page!
Look for "Update For Microsoft Windows ([KB5028254](https://support.microsoft.com/en-us/topic/july-26-2023-kb5028254-os-build-22621-2070-preview-66161ea6-c106-43e9-bef5-8410ccbd34e8))"

![image](https://github.com/valinet/ExplorerPatcher/assets/7243190/b9586708-ce55-46f3-b41d-5ed7837d9bf4)

**Reply by pyrates999** [reply]
KB5028254 matches the name of the preview update I suggested you uninstall.  Glad you found it though.

**Comment by Tiagoquix** [comment]
I can confirm that ExplorerPatcher became unstable on my Windows 11 after installing the KB5029263 update.

EDIT: The issues were resolved on my Windows 11 after deleting the keys `IsUndockedAssetAvailable` (located at `HKEY_CURRENT_USER\Software\ExplorerPatcher\twinui.pcshell`) and `OSBuild` (located at `HKEY_CURRENT_USER\Software\ExplorerPatcher`). After that, I restarted EP and about 3 minutes later I received a notification of symbols being downloaded. After that, I restarted EP again and had no more problems.

**Reply by vippium** [reply]
Yup @valinet, firstly sorry for tagging you. But it is critical on latest win11 stable update that EP is no more working. Please fix this critical bug ASAP. Thanks

I know that you are busy with your stuffs. But it is Critical. Issues are :
1. EP no more working on stable.
2. Action center is overlapping win10 taskbar.
3. Win10 taskbar crash on beta builds.

**Reply by lawrenmr** [reply]
Uninstalled this KB and EP stopped crashing, but a few mins later it re-installed automatically and after the next reboot, started crashing again - didn't stop Windows Update from runing, so uninstalled again - stops crashing and now stopped Windows Update for a month and will check back for an update to Explorer Patcher before resuming it

Update 18th Aug (after remving the KB and suspending Windows Update):
- Deleted the registry keys above
- Restarted ExplorerPatcher
- Waited for Symbols to download
- Restarted ExplorerPatcher
- Ran for a few days (not required, but that's what I did)
- Re-enabled Windows Update
- Re-installed KB5029263
- Rebooted
Now it appears stable, no more explorer crashes, so thanks for all of the help

**Reply by tallman69** [reply]
I was having the same issues this morning as well.  I tried pausing updates and uninstalling KB5029263 but it reinstalled automatically after rebooting.  EP somehow managed to stay working long enough for symbols for build 10.2621.2134 to be downloaded.  Now all it working as it should be.

**Reply by tpcr** [reply]
I have that update installed and working fine. I did uninstall EP and deleted the reg keys. After reinstalling EP it seemed to be working with all the updates. I am worried about my users in the field though.

**Reply by Leokuma** [reply]
@Vipin2003 Please don't be rude. This is a community project. Valinet doesn't make money out of this.

**Reply by vippium** [reply]
I'm not rude bro... Just wanna an update to fix overlap bug, so that i can use beta builds. I'm not able use EP in beta due to these critical bugs.

**Reply by pyrates999** [reply]
I don't think beta builds are a priority here, releases are.

**Reply by Tiagoquix** [reply]
The issues were resolved on my Windows 11 after deleting the keys `IsUndockedAssetAvailable` (located at `HKEY_CURRENT_USER\Software\ExplorerPatcher\twinui.pcshell`) and `OSBuild` (located at `HKEY_CURRENT_USER\Software\ExplorerPatcher`). After that, I restarted EP and about 3 minutes later I received a notification of symbols being downloaded. After that, I restarted EP again and had no more problems.

**Reply by eduardomcorrea** [reply]
Unfortunately didn't work for me.
Keep crashing when open any folder, but other functions are ok.
OS Build 22621.2134

Try to uninstall, delete registry and delete folder, reboot... did the steps... keep crashing

**Reply by tpcr** [reply]
I have same OS Build and works for me. Open the console in settings>advanced to make sure the symbol library is loaded. The console may give you some clues

**Reply by tpcr** [reply]
Also, if you are using open-shell, make sure that is updated too. Classic-shell needs to be updated to Open-Shell

**Reply by tpcr** [reply]
If all that fails. Try a new profile and see if problem persists. If so, run sfc /scannow in command prompt

**Comment by MMOStars** [comment]
Worked for me.

**Comment by tpcr** [comment]
Worked for me too, and the list of uninstall updates does not show the affecting updates in my list. That list is hit or miss.

**Comment by zaadstra** [comment]
There is a symbols update (restart) and that makes all working fine again.

**Reply by tpcr** [reply]
Yes, deleting the reg keys forces a symbols update. Then all works fine.

**Reply by zaadstra** [reply]
I did nothing, just started up W11.

**Comment by Undisputed00x** [comment]
**It didnt fix the issue** just updated to August cumulative update (KB5029263)  deleted reg keys and forced symbols to be downloaded there was no fix and i have to go back to Win10 taskbar

**Comment by ssateneth** [comment]
I tried the regedit changes as described by Tiagoquix on 4 computers with OS build 22621.2134 Win11 Pro for Workstations and KB5029263 (Cumulative update 2023-08)

Explorer continues to crash, only after a certain amount of time after explorer has loaded though.

**Comment by tpcr** [comment]
Everyone who has mentioned an issue did not say whether the symbol update occurred. Make sure that 'Enable symbols download' is on in the Advanced Properties. Since none of my computers or users have issues, I have to assume that if you are experiencing Explorer crashes, you have a wrong or left over setting somewhere. Try completely removing EP from your file system and registry. If not working there is probably something left over.

**Reply by ghost** [reply]
EP was crashing for me after the WU (not the first time, so I knew the problem was symbols). When I looked at my settings, "Enable symbols download" had become unchecked. I re-checked it, and terminated explorer.exe int he Task Manager, then restarted it via "Run New Task". After that I started EP and the symbols finally downloaded. No more problems.

**Reply by as232** [reply]
In my case, "Enable Symbols Download" was already checked. The problem perissts -- right-clicking the taskbar causes Explorer to crash and restart.

**Reply by ghost** [reply]
You need to find a way to trigger the symbols download. You could try turning the setting off and restarting EP and then turning it back on, resetting EP to default settings and restarting EP, checking and/or deleting the registry keys under `Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher` and then uninstalling and then reinstalling EP, etc.

**Comment by ssateneth** [comment]
After the regedit thing, I rebooted the computers, did NOTHING and waited for notification that symbols downloaded, then rebooted the computers again. I did not open any folders, or click any start menus or task bars after deleting the regedit things.

**Comment by eduardomcorrea** [comment]
These steps alleviate the problem, but do not correct it.

**Comment by tpcr** [comment]
What do you means by 'do not correct it' I don't see any issues here.

**Reply by eduardomcorrea** [reply]
This entire topic are about an issue

**Reply by tpcr** [reply]
I just wanted clarification on what you meant by 'do not correct it'. The steps seem to correct my issues with EP

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1828

Please close this one.

Possible fix: In ExplorerPatcher Properties, go to Advanced and tick "Do not change the taskbar context menu"

**Reply by eduardomloureiro** [reply]
this fix seems to have worked for me.

EP was crashing when right clicking the task bar (win 11 style);
I tried changing it to win 10 style, and it stopped crashing, but then win+tab and alt+tab stopped working.
now back to win 11 style, it has not crashed since ticking "Do not change the taskbar context menu".

thanks!

**Comment by aieiebrazor** [comment]
my file explorer continue crashing random when i try to open a folder. Installed latest EP upgrade but no result  : ( 
I use W10 Style
Any fix?
Tnx

**Reply by pyrates999** [reply]
Switch to windows 11 style taskbar for now.

**Reply by as232** [reply]
I am running Windows 11 Task bar and having the same problem. Enable Symbols Download was already checked as well.

**Reply by pyrates999** [reply]
Possible fix: In ExplorerPatcher Properties, go to Advanced and tick "Do not change the taskbar context menu"

**Comment by Jerryljy** [comment]
Win11 OS build 22621.2134 continue crash and restart when right click on taskbar, looks like not able to open it.
latest version 22621.1992.56.3 installed.
any clue to fix the issue?

**Reply by pyrates999** [reply]
Switch to windows 11 style taskbar for now.

**Reply by pyrates999** [reply]
Possible fix: In ExplorerPatcher Properties, go to Advanced and tick "Do not change the taskbar context menu"

**Reply by as232** [reply]
This causes EP not to crash, but it takes away the Win10 context menu, which was most of the reason I installed it to begin with. 

Un-ticking "Do not change the taskbar context menu" and restarting causes it to crash again on right-click, so there's some issue with that contet menu and the latest Win11 update apparently.

**Reply by LarsErikOsterud** [reply]
Running WIndows 11 22621.2215. Explorer Patcher kept restaring all the time yesterday.
But today it seems to work OK (but I did delete some registry entries).
Is there certain parts that do not work (so we can keep those off)?
I allready had the "Don't change taskbar context manu" checked.

**Reply by ghost** [reply]
You need to find a way to trigger the "symbols download". Check the comments in this thread for methods.

**Reply by ghost** [reply]
Same issue as as232 can't use Windows 11 taskbar unless "Do not change the taskbar context menu" is checked but then you loose most of the context menu. Unfortunately, all the fixes listed here do not work for me.

**Reply by as232** [reply]
Symbols download was already checked. Also tried un-checking it, then re-checking. No improvement, issue persists.

**Reply by pyrates999** [reply]
Never untick download symbols when you are running a public release of windows:

Possible fix: In ExplorerPatcher Properties, go to Advanced and tick "Do not change the taskbar context menu"

**Comment by igroglaz** [comment]
got same issue. nothing helps :( 
Windows: 10.0.22621 Build 22621

**Reply by pyrates999** [reply]
Possible fix: In ExplorerPatcher Properties, go to Advanced and tick "Do not change the taskbar context menu"

**Reply by igroglaz** [reply]
I've tried it; it doesn't work.

**Comment by Undisputed00x** [comment]
> Possible fix: In ExplorerPatcher Properties, go to Advanced and tick "Do not change the taskbar context menu"

Thats not a fix but a temporary workaround until there is an **actual fix**!!

**Comment by ghost** [comment]
The latest update fixed this problem for me. Thank you very much!
